driftbase/flexmatch.py

```diff
@@ -894,23 +936,48 @@ def __exit__(self, exc_type, exc_val, exc_tb):
     def is_banned(self) -> bool:
```

Comment:
How come is_banned doesn't take the list of matches into account?

Reply:
is_banned checks get_unban_date() where it checks num_bans.
I'll update it to check the list of matches instead.

Reply:
updated